portality/constants.py

```diff
@@ -53,7 +53,9 @@
 EVENT_APPLICATION_EDITOR_GROUP_ASSIGNED = "application:editor_group:assigned"
 EVENT_JOURNAL_ASSED_ASSIGNED = "journal:assed:assigned"
 EVENT_JOURNAL_EDITOR_GROUP_ASSIGNED = "journal:editor_group:assigned"
+EVENT_JOURNAL_DISCONTINUING_SOON = "journal:discontinuing_soon"
 
+NOTIFICATION_CLASSIFICATION_STATUS = "status"
```

Comment:
If we invent a new classification status, we need to think about how this is displayed in the notifications search UI.  It will need an icon, or at least to check the edges code that shows the notifications search that it falls back to the text gracefully

Reply:
no, I think a new one makes sense, though "status" is a bit of an overloaded term.  Perhaps "alert"?  Then the icon can be a bell or something.

Reply:
To do this, set the status in the constants like you have done, for this new "alert", then in `portality/static/js/edges/notifications.edge.js` add a new SVG icon in the `icons` dictionary, and a new description in the `classifications` dictionary and this should then display as appropriate